pkg/handlers/webhookreceiver_test.go

```diff
@@ -25,16 +28,13 @@ func (fn RoundTripperFunc) RoundTrip(r *http.Request) (*http.Response, error) {
 	return fn(r)
 }
 
-const (
-	dummyJWT = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhbGciOiJIUzI1NiIsInR5cCI6IkJlYXJlciJ9.0rLPJ-zaY_wFsADkvKmW5nsZzeyFmCP0276XSrkctb4"
-)
-
 var _ = Describe("Webhook Handlers", func() {
 
 	var (
 		mockCtrl               *gomock.Controller
 		mockClient             *clientmocks.MockClient
-		testConn               *sdk.Connection
+		mockUpdater            *clientmocks.MockStatusWriter
```

Comment:
not big deal, but it would be better if this names `mockStatusWriter` or `mockStatusUpdater`